pkg/logcli/client/client.go

```diff
@@ -226,13 +227,14 @@ func (c *DefaultClient) doRequest(path, query string, quiet bool, out interface{
 			if err := resp.Body.Close(); err != nil {
 				log.Println("error closing body", err)
 			}
+			respErrorMsg = string(buf)
 			continue
 		}
 		success = true
 		break
 	}
 	if !success {
-		return fmt.Errorf("Run out of attempts while querying the server")
+		return fmt.Errorf("Run out of attempts while querying the server,err: %v", respErrorMsg)
```

Comment:
In case line 219 returns an errors, `respErrorMsg` is `""`, leading to a weird message.

```suggestion
		msg := "Run out of attempts while querying the server"
		if respErrorMsg != "" {
		    msg = fmt.Sprintf("%s; response: %s", msg, respErrorMsg)
		}
		return fmt.Errorf(msg)		
```